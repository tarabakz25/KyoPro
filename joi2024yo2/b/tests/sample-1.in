5 1 3
10 1
40 1
30 1
20 1
50 1
1 2 4
1 3 5
1 1 5
