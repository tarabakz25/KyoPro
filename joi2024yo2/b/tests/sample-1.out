45
50
75
